libraries/kmp-iap/library/src/iosMain/kotlin/io/github/hyochan/kmpiap/openiap/WebhookTransport.ios.kt

```
@@ -0,0 +1,337 @@
+            activeTask = task
+            task.resume()
+            delegate.awaitFinished()
+            true
+        } finally {
+            // NSURLSession holds a strong reference to its delegate
+            // (the SseDelegate above) until the session is invalidated.
+            // Without this, every reconnect pass leaks the session +
+            // delegate + buffered NSMutableData chain.
+            session.finishTasksAndInvalidate()
+        }
+    } catch (error: Throwable) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)